(feat/status-go)  Add toggle for mainnet transaction validation with …

kounkou opened the pull request:
### Description

(feat/status-go)  Add toggle for mainnet transaction validation with nimbus

This PR adds the Advanced setting toggle for Nimbus enabling
It contains :

- Adds the Status-go code for persisting the toggle state in database
- Adds the migration files to make status-go actions possible
- Adds the new column for he Nimbus proxy toggle setting

This PR fixes [13204](https://github.com/status-im/status-desktop/issues/13204)

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | ada6944d | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4574/1/) | 2024-01-16 23:30:59 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240116-232724-ada694-pr4574.zip) |
| :heavy_check_mark: | ada6944d | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4574/1/) | 2024-01-16 23:32:38 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240116-232719-ada694-pr4574.zip) |
| :heavy_check_mark: | ada6944d | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4574/1/) | 2024-01-16 23:32:52 | ~5 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240116-232719-ada694-pr4574.aar) |
| :heavy_multiplication_x: | ada6944d | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/1/) | 2024-01-16 23:43:32 | ~16 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/1//consoleText) |
| :heavy_multiplication_x: | ada6944d | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/2/) | 2024-01-17 18:15:11 | ~20 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8fddae6d | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-4574/2/) | 2024-01-17 18:21:09 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-240117-181919-8fddae-pr4574.zip) |
| :heavy_check_mark: | 8fddae6d | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-4574/2/) | 2024-01-17 18:21:59 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-240117-181919-8fddae-pr4574.aar) |
| :heavy_multiplication_x: | 8fddae6d | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/3/) | 2024-01-17 18:26:17 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/3//consoleText) |
| :heavy_multiplication_x: | 8fddae6d | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/4/) | 2024-01-17 21:08:55 | ~19 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/4//consoleText) |
| :heavy_multiplication_x: | 8fddae6d | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/5/) | 2024-01-18 06:08:16 | ~19 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/5//consoleText) |
| :heavy_multiplication_x: | 8fddae6d | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/6/) | 2024-01-18 07:03:36 | ~32 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/6//consoleText) |
| :heavy_multiplication_x: | 8fddae6d | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/7/) | 2024-01-18 07:37:42 | ~32 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/7//consoleText) |
| :heavy_multiplication_x: | 8fddae6d | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/8/) | 2024-01-18 08:18:27 | ~36 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/8//consoleText) |
| :heavy_multiplication_x: | 8fddae6d | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/9/) | 2024-01-18 08:38:03 | ~19 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/9//consoleText) |
| :heavy_check_mark: | 8fddae6d | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/10/) | 2024-01-18 09:20:38 | ~37 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-4574/10//consoleText) |
| :heavy_check_mark: | 8fddae6d | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-4574/5/) | 2024-01-18 17:23:30 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-240118-171733-8fddae-pr4574.zip) |

kounkou commented on multiaccounts/settings/structs.go:
```diff
@@ -127,6 +127,7 @@ type Settings struct {
 	DeviceName                string           `json:"device-name"`
 	DisplayName               string           `json:"display-name"`
 	Bio                       string           `json:"bio,omitempty"`
+	IsImageDeleted            bool             `json:"is-image-deleted"`
```
This seems off. I'll regenerate files.

kounkou committed:
(feat/status-go) Add toggle for mainnet transaction validation with
nimbus

This PR adds the Advanced setting toggle for Nimbus enabling
It contains :

- Adds the Status-go code for persisting the toggle state in database
- Adds the migration files to make status-go actions possible
- Adds the new column for he Nimbus proxy toggle setting

kounkou commented:
This PR is not necessary anymore as the backend for handling the settings is implemented in https://github.com/status-im/status-go/pull/4254/files
